nursery/capture-screenshot-go.yml

```
@@ -0,0 +1,31 @@
+rule:
+  meta:
+    name: capture screenshot (Go)
+    namespace: collection/screenshot
+    description: Detects screenshot capability via WinAPI for Go files.
+    author:
+      - joakim@intezer.com
+    scope: file
+    att&ck:
+      - Collection::Screen Capture [T1113]
+    mbc:
+      - Collection::Screen Capture::WinAPI [E1113.m01]
+  features:
+    - and:
+      - match: compiled with Go
+      - or:
+        - and:
+          - string: syscall.NewLazyDLL # Dynamic loading of DLLs
```

can use a format like this so that the comment will show up in UI/output.

```suggestion
          - string: syscall.NewLazyDLL
            description: Dynamic loading of DLLs
```